src/MSBuild/CommandLineSwitches.cs

```diff
@@ -222,6 +223,7 @@ private struct ParameterizedSwitchInfo
             new ParameterlessSwitchInfo(  new string[] { "filelogger8", "fl8" },                     ParameterlessSwitch.FileLogger8,           null),
             new ParameterlessSwitchInfo(  new string[] { "filelogger9", "fl9" },                     ParameterlessSwitch.FileLogger9,           null),
             new ParameterlessSwitchInfo(  new string[] { "distributedfilelogger", "dfl" },           ParameterlessSwitch.DistributedFileLogger, null),
+            new ParameterlessSwitchInfo(  new string[] { "analyzeStats", "as" },                     ParameterlessSwitch.AnalyzeStats,          null),
```

Comment:
nit: There was a decision not to use the short alias: https://github.com/dotnet/msbuild/blob/25b5f75df908bf749b1283d76b37996c6a02bb67/src/MSBuild/CommandLineSwitches.cs
Just wondering if we want to apply the decision here as well? @JanKrivanek what do you think?